2025 site copy updates (#2267)
writethedocs/www: docs/conf/portland/2025/convince-your-manager.md

---
template: {{year}}/generic.html

> og:image: _static/conf/images/headers/{{shortcode}}-{{year}}-opengraph.jpg

Reviewer: Replace with image, https://drive.google.com/file/d/1_EgNvTCjQEG3yCrz3MGkQrbwBSwaxEbI/view?usp=drive_link

---

> # Convince Your Employer

Reviewer: ```suggestion
# Convince Your Manager
```

> Do you need help justifying why your employer should send you to Write the Docs? Do you need help convincing your community to submit a Writing Day Project? Don’t worry – you’re not alone. Based on the experiences of some of our previous attendees, we’ve put together two sample emails and a list of resources below. Feel free to adapt and share with your employer and community to convince them that there are many benefits of attending!

Reviewer: ```suggestion
Do you need help justifying why your manager should send you to Write the Docs? Do you need help convincing your community to submit a Writing Day Project? Don’t worry – you’re not alone. Based on the experiences of some of our previous attendees, we’ve put together two sample emails and a list of resources below. Feel free to adapt and share with your employer and community to convince them that there are many benefits of attending!
```

> ## Sample email

Reviewer: ```suggestion
## Sample Email #1

### Convince your manager to send you to Write the Docs
```

I'm unsure how to add extra lines.

Remember to change the things in [brackets]!

*FROM: [your name]*

*TO: [your employer or manager’s name]*

*SUBJECT: Professional Development: Documentation Community Conference*

*I’d like to attend Write the Docs Portland from May 4-6, 2025. This three-day event explores the art and science of documentation, and covers a diverse range of topics related to documentation in the software industry.*

*Write the Docs conferences bring together everyone who writes the docs – Tech Writers, Developers, Developer Relations, Customer Support – making the events an ideal networking opportunity. Each conference successfully combines a number of different event formats to deliver engaging, practical, and timely content.*

> *There is a single track of talks, a parallel unconference event, and a community writing day. The sessions from last year will give you a good idea of the kinds of topics covered, many of which are relevant to my work.*

Reviewer: ```suggestion
*There is a single track of talks, a parallel Unconference event, and a community writing day. The sessions from last year will give you a good idea of the kinds of topics covered, many of which are relevant to my work.*
```

> *There is a single track of talks, a parallel unconference event, and a community writing day. The sessions from last year will give you a good idea of the kinds of topics covered, many of which are relevant to my work.*

Reviewer: ```suggestion
*There is a single track of talks, a parallel unconference event, and a community writing day. The [sessions from last year](https://www.writethedocs.org/conf/portland/2024/speakers/) will give you a good idea of the kinds of topics covered, many of which are relevant to my work.*
```

*Costs:*

> - *Conference ticket - $650*

Reviewer: ```suggestion
- *[Conference ticket](https://www.writethedocs.org/conf/portland/2025/tickets/) - $650*
```

*Benefits:*

- *Discovering solutions to problems I’m facing at work*
- *Exposure to the latest ideas, techniques, and tools for software docs*
- *Opportunity to learn from the best doc teams in the industry*
- *Building professional connections with other documentarians*

*Thanks in advance, [your name]*

> ## Corporate Resources

Reviewer: ```suggestion
### Corporate Resources
```

When discussing how to pitch the conference, a few generally helpful tips emerged:

- Highlight specific talks that relate to ongoing projects at work (after talk line up has been announced).
- If your company is looking to hire another documentarian, the networking is an excellent resource.
- Attending raises the visibility of your company in the community. 
- Establishes your team's reputation for caring about their docs.
- Share info about previous conferences. View the [main page overview](https://www.writethedocs.org/conf/portland/2025/) explaining Write the Docs, history, and schedule overview.

> ## Convince your employer or community to participate in Writing Day

Reviewer: ```suggestion
## Sample Email # 2

### Convince your manager or community to participate in Writing Day
```

> ### Sample email

Reviewer: ```suggestion
```

Remember to change the things in [brackets]!

*FROM: [your name]*

*TO: [your employer or community lead's name]*

*SUBJECT: Docs hackathon: Documentation Community Conference*

*I’d like to attend Write the Docs Portland from May 4-6, 2025. This three-day event explores the art and science of documentation, and covers a diverse range of topics related to documentation in the software industry. I am particularly interested in leading a project at Writing Day. I believe this event can positively impact our project and community.*

*Writing Day is modeled after the concept of “code sprints” or “hackathons”, which are common in open-source conferences. The primary goal is to bring interesting individuals into the same room and have them work towards a shared goal; in this case the goal is creating or improving documentation or other related projects.*

*Costs:*

> - *[Conference tickets](https://www.writethedocs.org/conf/portland/2025/tickets/)*

Reviewer: ```suggestion
- *[Conference ticket](https://www.writethedocs.org/conf/portland/2025/tickets/) - $650*
```
